src/commands.ts

```diff
@@ -462,3 +468,11 @@ async function hostsCMD(shell: Shell) {
     }
     shell.t.writeln(res)
 }
+
+async function closeCMD(shell: Shell) {
```

Comment:
There can be more then one connection at a time.  Please add and args use activeG if it's missing.